webpack/webpack5-load-themed-styles-loader/src/test/LoadThemedStylesLoader.test.ts

```diff
@@ -2,12 +2,28 @@
 // See LICENSE in the project root for license information.
 
 import * as loader from '..';
+import type { Stats } from 'webpack';
 import LoadThemedStylesMock = require('./testData/LoadThemedStylesMock');
 import getCompiler from './testData/getCompiler';
+import { Path } from '@rushstack/node-core-library';
 
 const MATCH_GENERATED_LOADER_STRING_REGEXP: RegExp = /var\sloader\s\=\srequire\(["'](.+?)["']\)/;
 const MATCH_LOADER_DOT_LOADSTYLES_FUNCTION_ASYNC_VALUE_REGEXP: RegExp = /loader\.loadStyles\(.+?,\s(.+?)\)/;
 
+function getLoadThemedStylesLibPath(stats: Stats): string {
+  // Example:
+  //   var loader = require("C:\\Git\\rushstack\\libraries\\load-themed-styles\\lib\\index.js");
+  const content: string = stats.toJson({ source: true }).modules?.[0].source?.toString() ?? '';
+  const match: RegExpExecArray | null = MATCH_GENERATED_LOADER_STRING_REGEXP.exec(content);
+  // Example:
+  //   C:\\Git\\rushstack\\libraries\\load-themed-styles\\lib\\index.js
+  const escapedPath: string = match?.[1] ?? '';
+  // Example:
+  //   C:\Git\rushstack\libraries\load-themed-styles\lib\index.js
+  const loadThemedStylesLibPath: string = JSON.parse(`"${escapedPath}"`);
```

Comment:
@TheLarkInn FYI these new tests were failing on Windows. The reason is that your regex extracts `a\\b\\c` from a string literal expression like `"a\\b\\c"`, so the backslashes are escaped. Then the string comparison fails when comparing to a normal string `a\b\c`.  To be 100% correct we need to unescape using a full JavaScript evaluator, but `JSON.parse()` is a close enough approximation for typical ASCII characters.

I also replaced `expect(x).toEqual(y)` with a file path comparison.  This might avoid other spurious differences such as case-insensitive filesystems or `x/./y`, but it weakens the assertion to disregard differences in absolute versus relative paths. Let me know if that is a problem.

Reply:
LGTM